.github/workflows/build-build-develop.yml

```diff
@@ -0,0 +1,79 @@
+name: Build develop packages
+on:
+  push:
+    branches:
+    - develop
+
+jobs:
+  build-deb:
+    if: github.repository == 'CodeForPoznan/alinka-pyside'
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v4
+    - name: build working docker image
+      run: |
+        make build
+    - name: get application version
+      # for unknown (yet?) reason setting APP_VERSION
+      # before `make build` causes Github action to fail
+      # with `make: docker: Argument list too long` error
+      run : |
+        APP_VERSION=$(docker compose run --rm app poetry version --short)-$(git rev-parse --short ${{ github.sha }})
+        INSTALLER_FILE_NAME="alinka-${APP_VERSION}.deb"
+        echo "APP_VERSION=$APP_VERSION" >> $GITHUB_ENV
+        echo "INSTALLER_FILE_NAME=$INSTALLER_FILE_NAME" >> $GITHUB_ENV
+    - name: build distribution package
+      # we don't use currently Makefile here
+      # as we want to inject git has as version
+      # instead of relying on `poetry version`
+      run: |
+        docker compose run --rm -u root app pyinstaller alinka.spec --noconfirm
+        docker compose run --rm -u root app ./package.sh
+        docker compose run --rm -u root app fpm -v ${{ env.APP_VERSION }} -p ${{ env.INSTALLER_FILE_NAME }}
+    - name: publish distribution package
+      uses: actions/upload-artifact@v4
+      with:
+        name: ${{ env.INSTALLER_FILE_NAME }}
+        path: ${{ env.INSTALLER_FILE_NAME }}
+
+  build-win:
+    if: github.repository == 'magul/alinka-pyside'
```

Comment:
Is this intendet? `'magul/alinka-pyside'`

Reply:
No, well caught! I will update it to `CodeForPoznan`.